### ISSUE TITLE
Alt text, titles for images are not being rendered in anyway

### ISSUE THREAD

**Issue by KenLFG** [coolwanglu/pdf2htmlEX#716]
I know that images on a particular page are effectively flattened into a single background image but is there no way to have the alt text included for those images rendered for accessibility? All it would need to be is an absolute positioned div tag the same size and location of the image with a title. I was hoping there was some setting I was missing but I don't think I am, am I?
